sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureStorageFileAttributeView.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.nio;
+
+import java.nio.file.attribute.FileAttributeView;
+
+/**
+ * Provides support for properties specific to Azure Blob Storage such as tier.
+ *
+ * {@inheritDoc}
+ */
+public class AzureStorageFileAttributeView implements FileAttributeView {
+
```

Comment:
This type was created in this PR so that the AzureFileStore type can accurately say it is supported, but it will be given actual functionality in a later PR when setting blob/file attributes is added.